python/apps/taiga/src/taiga/comments/mixins.py

```diff
@@ -9,11 +9,9 @@
 from taiga.comments.models import Comment
 
 
-class CommentsMixin(models.Model):
+class StoryCommentsMixin(models.Model):
```

Comment:
The name should be something like `RelatedCommentsMixin`, isn't it?

Reply:
Yes, it's to list the other relation's end. I'll use that name